hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HsyncGenerator.java

```diff
@@ -126,9 +116,6 @@ public Void call() throws Exception {
       for (FSDataOutputStream outputStream : outputStreams) {
         outputStream.close();
       }
-      for (FileSystem fs : fileSystems) {
```

Comment:
I don't recall why we wanted to have multiple file system objects.
Was it to have multiple RpcClient and XceiverClientRatis objects to parallelize hsync clients? @duongkame

Reply:
I think this is added due to my suggestion https://github.com/apache/ozone/pull/6925/commits/1d273dffe468b90956bafb1b0a2dffd93be867cf, but I think a since FileSystem cache is not disabled, it will still use the same FileSystem instance anyway.